LICENSE

```diff
@@ -175,7 +175,7 @@
 
    END OF TERMS AND CONDITIONS
 
-   Copyright 2021 Tinkoff Bank
+   Copyright 2021 T-Bank
```

Comment:
Wasn't it decided not to do this renaming in LICENSE?

Reply:
Hmm, forgot to pull, sorry. Fixed in newest commit

Reply:
It seems like it wasn't fixed by the newest commit. The most reliable source of a change is a tab "Files changed" in the PR.

Reply:
Well, now :Files changed" tab does not even show the change in LICENSE. But I guess I fixed my mistake, now it surely has the correct version